tests/pytests/server.py

```diff
@@ -21,12 +21,34 @@
 sys.path.insert(0, "")  # for running from cmake
 
 
+def make_backends(backends_count, group_id):
+    '''
+    Returns configuration of a single group that has @backends_count backends.
+    '''
+    backends = []
+    for i in xrange(backends_count):
```

Comment:
`i` here and `node`[there](https://github.com/interiorem/elliptics/pull/112/files#diff-dcb2340e644831a7140b6d308d00fadfR40) are useless and can be replaced by `_`.